cdc/redo/writer/file.go

```diff
@@ -182,6 +182,12 @@ func NewWriter(ctx context.Context, cfg *FileWriterConfig, opts ...Option) (*Wri
 		w.uuidGenerator = uuid.NewGenerator()
 	}
 
+	err := os.MkdirAll(cfg.Dir, common.DefaultDirMode)
```

Comment:
should you check whether cfg.Dir is empty or cfg.Dir exists

Reply:
Good point. If `cfg.Dir` is not empty, maybe there exist some dirty files which are not cleaned during changefeed removing. I will add the `cfg.Dir` check.

Reply:
Oh, the check may have some issues here. `cfg.Dir` is a public dir for meta/ddl/row log writing. If meta writing already created a dir, then the dir is not empty at all.

Reply:
check the condition cfg.Dir = " " ?